docs/pages/develop/development-builds/development-workflows.mdx

```diff
@@ -36,7 +36,7 @@ To get your project's ID, use the URL in the [app config's `expo.updates.url`](/
 
 {/* TODO: @aman move this section out of this page -- make it part of the main path or a standalone guide, this is important part that should be searchable for eg: Deep linking URLs for Development builds/EAS Builds (when using expo-dev-client) */}
 
-### Deep linking URLs
+### Deep linking to an update's URL
```

Comment:
What does update signify here? I don't think we signify it in this section.

Reply:
IMO "Deep linking URL's" could mean _any_ deep link, but the original text is only about a very specific deep link that opens an update URL.

Of course, I kind of break this by then putting info about deep links in general as a sub-heading under this. I could move that subsection up a level.

Reply:
Thanks for the clarifixation. It has been a while since I've looked at this doc, I was lacking context. I think the title and the sub section are both fine.